projects/com.oracle.truffle.llvm.test/src/com/oracle/truffle/llvm/test/Constants.java

```diff
@@ -31,6 +31,9 @@
 
 public class Constants {
 
+    public static final boolean USE_BINARY_PARSER = false;
```

Comment:
Could we have that as a `LLVMBaseOption`? Then you and others can test the binary parser without having to modify the source code.